templates/OperationSnippet.stencil

```diff
@@ -5,13 +5,13 @@ public func {{ op.name }}(
     {% endfor %}
     withOptions options: {{ model.name }}Options? = nil,
     completionHandler: @escaping HTTPResultHandler<{{ op.returnType.name }}>
-) {
+) throws {
```

Comment:
Hmm... if the method body didn't already throw, we shouldn't make it throw. Instead of throwing AzureErrors, we should probably log an error and early return. We had UX study feedback that the more things that are throwable the more confusing it is for customers, so this is actually a big usability change.